wpilibcExamples/src/main/cpp/examples/Frisbeebot/cpp/Robot.cpp

```diff
@@ -5,7 +5,9 @@
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
-#include <frc2/commands/CommandScheduler.h>
+#include "../include/Robot.h"
```

Comment:
uhh... This does not look right.  It should pick up that the include directory is in fact an include directory.

Reply:
Take a look at the gears bot example.

Reply:
VSC's intellisense goes absolutely batshit if you don't do this, unfortunately.

Reply:
@ThadHouse Thoughts?

Reply:
I am very much against this. `#include "Robot.h"` is the correct include to use here, regardless of what VSC's intellisense says.

Reply:
This could be incorrect in VS Code when the example is used, and is a very poor idea. It just needs to be `#include "Robot.h"`. Ignore VS Code intellisense issues when building allwpilib, I've never been able to get them working, and its not that big of an issue especially when the build works.